ASPNetCoreMastersTodoList.Api/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace ASPNetCoreMastersTodoList.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ErrorController : ControllerBase
+    {
+        [HttpGet()]
```

Comment:
please retain the `[Route("/error")]` in your session 3 PR, there is no need to change. `[HttpGet]` is not needed and may prevent error handling for other http verbs

Reply:
hi @jegtugado pag binabalik ko siya sa route("/error") may error na ako sa swagger kasi siguro naka mapcontrollers na lng ako sa startup.